documentation/asciidoc/topics/proc_disabling-fips-mode-cr.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: PROCEDURE
+
+[id="proc_disabling-fips-mode-cr_{context}"]
+= Disabling FIPS mode in your {brandname} CR
+
+[role="_abstract"]
+For {brandname} {ispn_version}, an issue exists when you want your {brandname} instance to run on any {openshiftshort} cluster that has Federal Information Processing Standards (FIPS) mode enabled. {brandname} {ispn_version} does not support FIPS mode, so you must disable FIPS mode in your {brandname} CR.
+
+After you disable FIPS mode in your {brandname} CR, any mechanism that uses a JVM, such as {brandname} server, {brandname} CLI, or Gossip router, ignores FIPS mode. This happens, because the JVM no longer loads FIPS-related cryptographic libraries on implementation startup. 
+
+[NOTE]
+====
+If an {openshiftshort} cluster has FIPS mode disabled, you do not need to explicitly disable FIPS mode in your {brandname CR} configuration.
+====
+
+.Prerequisites
+* Created a {brandname} CR on {openshiftshort}, so that your {brandname} Operator can interact with {openshiftshort}. 
```

Comment:
Use {ispn_operator} instead of {brandname} Operator
This attribute is defined in both community and downstream attributes file